### PR TITLE
New package: Snes9x

### PR DESCRIPTION
**Pull request by Vlad-Romanov**
will be adding snes9x-gtk front-end package soon too

### CODE REVIEW

File: srcpkgs/snes9x/template [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Template file for 'snes9x'
+pkgname=snes9x
+version=1.56.2
+revision=1
+build_wrksrc="unix"
```
Comment (maxice8) [inline srcpkgs/snes9x/template]: doesn't need to be quoted

File: srcpkgs/snes9x/template [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+build_style=gnu-configure
+configure_args="--enable-netplay"
+hostmakedepends="intltool nasm autogen autoconf automake"
+makedepends="minizip-devel pulseaudio-devel libpng-devel SDL-devel SDL2-devel libXv-devel
+ xorg-server-devel zlib-devel gtk+3-devel"
+depends="minizip libpulseaudio pulseaudio desktop-file-utils libpng SDL SDL2 adwaita-icon-theme
```
Comment (maxice8) [inline srcpkgs/snes9x/template]: don't need to add stuff that is detected via shlibs

File: srcpkgs/snes9x/template [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+ libXv gtk+3 zlib"
+short_desc="A Super Nintendo emulator in active development"
+maintainer="VladTheImpaler <vladimir-romanov-ii@yandex.com>"
+license="LGPL"
```
Comment (maxice8) [inline srcpkgs/snes9x/template]: not a spdx short-identifier

File: srcpkgs/snes9x/template [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+homepage="http://www.snes9x.com/"
+distfiles="https://github.com/${pkgname}git/${pkgname}/archive/${version}.tar.gz"
+checksum=1557ade36c3b47aedf766b4504d141521f565664a6264c04f4c44f5a2dfada4f
+
+pre_configure() {
+
```
Comment (maxice8) [inline srcpkgs/snes9x/template]: remove extra blank lines

File: srcpkgs/snes9x/template (hunk 2 above)
Comment (Vlad-Romanov) [reply srcpkgs/snes9x/template]: Which specifically is detect via shlibs? Sorry this is first day I have done contributions

File: srcpkgs/snes9x/template (hunk 3 above)
Comment (Vlad-Romanov) [reply srcpkgs/snes9x/template]: Okay, I will look up correct one, but this one is currently in use with snes9x-gtk package which is why is there

**Pull request closed without merging**